stdlib/auth/src/main/ballerina/src/auth/outbound_basic_auth_provider.bal

```diff
@@ -55,25 +59,26 @@ public type OutboundBasicAuthProvider object {
     # Inspects the incoming data and generates the token for Basic authentication.
     #
     # + data - Map of the data, which is extracted from the HTTP response.
-    # + return - The String token, the `Error` occurred when generating the token, or `()` if nothing is to be returned.
+    # + return - The token as a `string`, or an `Error` if occurred when generating the token, or `()`
```

Comment:
```suggestion
    # + return - The token as a `string`, or an `auth:Error` if occurred when generating the token, or `()`
```

Reply:
Wrong return statement format. Please refer the template.